topbin/dune

```diff
@@ -28,12 +28,25 @@
  (libraries coq.toplevel)
  (link_flags -linkall))
 
+(rule
+ (targets coqc_byte_bin.ml)
+ (action (copy# coqc_bin.ml %{targets})))
+
+(executable
+ (name coqc_byte_bin)
+ (public_name coqc.byte)
+ (package coq)
+ (modules coqc_byte_bin)
+ (libraries coq.toplevel)
+ (modes byte)
+ (link_flags -linkall))
```

Comment:
Isn't having the .bc target enough? If want to install it, then feel free to add an appropriate `(install ..)` stanza

Reply:
Indeed thanks, I dunno what I was thinking....